sal_interfaces/SALSubsystems.xml

```diff
@@ -639,14 +639,14 @@
   <IndexEnumeration>no</IndexEnumeration>
   <Generics>yes</Generics>
   <ActiveDevelopers>CCS Team</ActiveDevelopers>
-  <Github>http://github.com/lsst-camera-ccs</Github>
-  <JenkinsTestResults></JenkinsTestResults>
-  <RubinObsContact></RubinObsContact>
-  <CSCDocs></CSCDocs>
-  <ProductOwner></ProductOwner>
+  <Github>http://github.com/lsst-camera-ccs/</Github>
+  <JenkinsTestResults>https://srs.slac.stanford.edu/releaseManagement/</JenkinsTestResults>
+  <RubinObsContact>Tony Johnson</RubinObsContact>
+  <CSCDocs>https://confluence.slac.stanford.edu/display/LSSTCAM/CCS+Subsystem+Manuals</CSCDocs>
+  <ProductOwner>Michael Reuter</ProductOwner>
   <RelatedDocuments>LSE-71</RelatedDocuments>
   <SoftwareLanguage>Java</SoftwareLanguage>
-  <RuntimeLanguages>IDL</RuntimeLanguages>
+  <RuntimeLanguages>IDL,Java</RuntimeLanguages>
   <VendorContact>Not Applicable</VendorContact>
   <Simulator>No</Simulator>
```

Comment:
This should be "Not Provided" or "Not Required"

Reply:
I changed my answer to "Internal to CSC" which I think is a more accurate answer given the allowed choices.